content/components/tab-nav.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Tab nav
+description: Tab nav contains a set of links that let users switch between views in the same context.
+reactId: tab_nav
+rails: https://primer.style/view-components/components/alpha/tabnav
+componentId: tab_nav
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=20366-72067&t=kKGC85STU5bcL0zg-4
+---
+
+import ComponentLayout from '~/src/layouts/component-layout'
+export default ComponentLayout
+
+<img
+  width="960"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/228976777-6239662c-0926-4e13-be0d-be1a1eff08ed.png"
+/>
+
+## Usage
+
+A tab nav is used to organize related content into different views within the same context.
+
+### Best practices
+
+- One of the tabs should be selected by default when the user loads the page.
+- Keep tab labels as short as possible - ideally no more than three words.
+- Avoid overwhelming the user with too many tabs. Instead consider a navigation structure like a [nav list](https://primer.style/design/components/nav-list) that's meant to handle more links.
+- If a user navigates to a different tab panel, form data should not be automatically saved or submitted. Changes must be explicitly be saved or submitted.
+- Don't nest a tab nav in another tab nav panel.
+- Don't force users to switch between multiple tabs to compare info. Each tab panel should have all of the information a user needs to complete their task.
+- Don't nest buttons or links in a tab.
```

Comment:
can we clarify what this means? we often have links behind tab content (thinking our pull request local nav/UI as an example)

Reply:
I'm referring to the tab itself, not the tab content. Any ideas how we might make that clearer? I'm having a hard time wording this...